.github/workflows/firebase.yml

```diff
@@ -38,3 +38,14 @@ jobs:
       env:
         FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
         PROJECT_ID: zebra-doc-internal
+    - name: Build book
+      run: |
+        mdbook build book/
+    - name: Deploy external book to firebase
+      uses: w9jds/firebase-action@v1.5.0
+      with:
+        args: deploy
+      env:
+        FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+        PROJECT_PATH: book/
+        PROJECT_ID: zebra-book
```

Comment:
This seems reasonable to me -- since testing this requires merging the PR, I think it would be reasonable to merge this and then make as many little fixup PRs as required to fix whatever problems show up.

Reply:
its already tested, I added `book` as the valid branches for this CI, let me restart CI now that dns isn't broken and you'll see the link is already active and deployed too

Reply:
actually @hdevalence the link is already good from previous deploys,

Reply:
Ah, should we change the branch to be `main` instead of `book`?

Reply:
its currently set to both, but we should delete the `book` branch trigger right before we merge it because it will no longer be necessary.